# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1404266115 14400
#      Tue Jul 01 21:55:15 2014 -0400
# Branch stable
# Node ID 534511deb10eaafc2f3138dd0aabd3ce322ccf28
# Parent  ed41db0317524c7c45e12dfde8827cf40dca6555
Use %!testif HAVE_CHOLMOD for sparse chol test added in cset 161b11e7da2d

* chol.cc: Use %!testif HAVE_CHOLMOD for chol %!test on sparse data that
was added in cset 161b11e7da2d.

diff --git a/libinterp/dldfcn/chol.cc b/libinterp/dldfcn/chol.cc
--- a/libinterp/dldfcn/chol.cc
+++ b/libinterp/dldfcn/chol.cc
@@ -370,17 +370,17 @@ sparse matrices.\n\
     }
 
   return retval;
 }
 
 /*
 %!assert (chol ([2, 1; 1, 1]), [sqrt(2), 1/sqrt(2); 0, 1/sqrt(2)], sqrt (eps))
 %!assert (chol (single ([2, 1; 1, 1])), single ([sqrt(2), 1/sqrt(2); 0, 1/sqrt(2)]), sqrt (eps ("single")))
-%!test   
+%!testif HAVE_CHOLMOD
 %! ## Bug #42587
 %! A = sparse ([1 0 8;0 1 8;8 8 1]);
 %! [Q, p] = chol (A);
 %! assert (p != 0);
 
 %!error chol ()
 %!error <matrix must be positive definite> chol ([1, 2; 3, 4])
 %!error <requires square matrix> chol ([1, 2; 3, 4; 5, 6])
